examples/training/torchtitan/README.md

```diff
@@ -32,39 +40,36 @@ The provided `torchtitan.yaml` configuration:
 - Sets up a 2-node cluster with 8 H100 (or H200) GPUs per node
 - Installs PyTorch nightly and TorchTitan requirements
 - Downloads the Llama 3.1 tokenizer
-- Runs distributed training using torchrun
-
-### Customizing the configuration
-
-You can override various parameters without editing the YAML file:
-
-```bash
-# Use 4 nodes and train a larger model
-sky launch -c torchtitan-multinode torchtitan.yaml \
-   --num-nodes 4 \
-   --env HF_TOKEN \
-   --env CONFIG_FILE=./torchtitan/models/llama3/train_configs/llama3_70b.toml # relative to the torchtitan's repo
-```
 
 ## Available model configurations
 
-TorchTitan includes pre-configured training recipes for:
+TorchTitan includes [pre-configured training recipes](https://github.com/pytorch/torchtitan/tree/main/torchtitan/models/llama3/train_configs) for:
 - Llama 3.1 8B: `llama3_8b.toml`
 - Llama 3.1 70B: `llama3_70b.toml`
 - Llama 3.1 405B: `llama3_405b.toml`
 
 Each configuration file specifies model architecture, parallelism strategies, and training hyperparameters optimized for different scales.
 
+## Scaling Up
+
+Launch parallel experiments with [Managed Jobs](https://docs.skypilot.co/en/latest/running-jobs/many-jobs.html):
```

Comment:
This does not have anything TODO with the following commands?

Reply:
oh right, I'll fix